docs/whatsnew/7.1.rst

```diff
@@ -45,6 +45,52 @@ is now supported by the Unified File Read/Write Interface.
 Note that this format is compatible with ``Table`` objects with integer, float,
 and/or string data types, but not ``QTable`` objects or other data types.
 
+.. _whatsnew-7.1-cosmology:
+
+Cosmology
+=========
+
+
+The `astropy.cosmology.traits` module hosts various parts of cosmologies, such as the
+:class:`~astropy.cosmology.traits.ScaleFactor` or
+:class:`~astropy.cosmology.traits.TemperatureCMB`. These traits can be used to more
```

Comment:
There should be a link to the glossary entry.

Reply:
Done!

Reply:
Done!